frontend/sass/pages/_admin.scss

```diff
@@ -196,3 +196,111 @@
     }
   }
 }
+
+.admin-panel-status-suspended {
+  color: map.get(map.get(abstracts.$highlight-colors, "pink"), "default");
+  margin-left: 5px;
+}
+
+.admin-panel-status-active {
+  color: map.get(map.get(abstracts.$highlight-colors, "cyan"), "default");
+  margin-left: 5px;
+}
+
+.admin-panel-report-label {
+  display: none;
+}
+
+.admin-panel-reports-grid {
+  min-width: fit-content;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 300px 1fr;
```

Comment:
Laittamalla tähän sekä headeriin `grid-template-columns: 75px 1fr 1fr 1fr 2fr 100px;` ja laittamalla h4-otsikoille h5-tyylin `className="h5"`lla mahtuu kaikki ihan kivasti rinnakkain ja ei mene mikään päällekkäin kun kaventaa. Labeleille vois laittaa samalla h5-tekstikoon.